<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LaWidget</class>
 <widget class="QDialog" name="LaWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>745</width>
    <height>347</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="hLauout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="dispLayout">
     <item>
      <widget class="PlotDisplay" name="plotDisplay" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="QLabel" name="autoLabel_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string notr="true">Trigger:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="triggerBox">
         <item>
          <property name="text">
           <string>Ch 1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ch 2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ch 3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ch 4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ch 5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ch 6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ch 7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ch 8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Condition</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Conds: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="condt1">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cond1">
         <item>
          <property name="text">
           <string>N</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>L</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>R</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>H</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>F</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="condt2">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cond2">
         <item>
          <property name="text">
           <string>N</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>L</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>R</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>H</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>F</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="condt3">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cond3">
         <item>
          <property name="text">
           <string>N</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>L</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>R</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>H</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>F</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="condt4">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>4</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cond4">
         <item>
          <property name="text">
           <string>N</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>L</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>R</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>H</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>F</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="condt5">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cond5">
         <item>
          <property name="text">
           <string>N</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>L</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>R</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>H</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>F</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="condt6">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>6</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cond6">
         <item>
          <property name="text">
           <string>N</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>L</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>R</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>H</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>F</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="condt7">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>7</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cond7">
         <item>
          <property name="text">
           <string>N</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>L</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>R</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>H</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>F</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="condt8">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>8</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cond8">
         <item>
          <property name="text">
           <string>N</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>L</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>R</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>H</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>F</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="Line" name="line_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>4</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <widget class="QLabel" name="autoLabel_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string notr="true">Time Div</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="timeDivDial">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="wrapping">
          <bool>true</bool>
         </property>
         <property name="notchesVisible">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="timeDivBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>7</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="prefix">
          <string notr="true"/>
         </property>
         <property name="suffix">
          <string notr="true"> ms</string>
         </property>
         <property name="decimals">
          <number>0</number>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <widget class="QLabel" name="autoLabel_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string notr="true">Time Pos</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="timePosDial">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="wrapping">
          <bool>true</bool>
         </property>
         <property name="notchesVisible">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="timePosBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>7</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="prefix">
          <string notr="true"/>
         </property>
         <property name="suffix">
          <string notr="true"> ms</string>
         </property>
         <property name="decimals">
          <number>0</number>
         </property>
         <property name="minimum">
          <double>-1000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <widget class="QLabel" name="autoLabel_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>70</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string notr="true">Volt Div</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="voltDivDial">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="wrapping">
          <bool>true</bool>
         </property>
         <property name="notchesVisible">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="voltDivBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>7</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="prefix">
          <string notr="true"/>
         </property>
         <property name="suffix">
          <string notr="true"> ms</string>
         </property>
         <property name="decimals">
          <number>0</number>
         </property>
         <property name="maximum">
          <double>1000.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotDisplay</class>
   <extends>QWidget</extends>
   <header>plotdisplay.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
